jdaviz/components/plugin_add_results.vue

```diff
@@ -52,7 +52,7 @@
 
     <slot></slot>
 
-    <v-row v-if="auto_update_result !== undefined">
+    <v-row v-if="false && auto_update_result !== undefined">
```

Comment:
Is `&&` even needed here?

Reply:
as opposed to just having `false` (or removing it or commenting the whole block out)?  Any of those are options, sure.

Reply:
How about I just remove the block and we can always revert from this diff when we want to reintroduce it?

Reply:
If you are going to reintroduce it soon, I'd say go for the easiest route but leave a comment. I'll leave the final choice to you. Thanks!

Reply:
see latest commit for commenting out this block (and removing the `false`)